src/components/gateway/GatewayBackendsPage.tsx

```
@@ -0,0 +1,284 @@
+  }>(resources);
+
+  const gatewayWatch = watchedResources.gateway;
+  const httpRoutesWatch = watchedResources.httpRoutes;
+
+  const allLoaded = gatewayWatch.loaded && httpRoutesWatch.loaded;
+
+  const loadErrors = [gatewayWatch.loadError, httpRoutesWatch.loadError].filter(Boolean);
+  const loadError =
+    loadErrors.length > 0
+      ? new Error(loadErrors.map((err) => err.message).join('; '))
+      : null;
```

ESLint

[error] Replace `⏎······?·new·Error(loadErrors.map((err)·=>·err.message).join(';·'))⏎·····` with `·?·new·Error(loadErrors.map((err)·=>·err.message).join(';·'))`

(prettier/prettier)